common/djangoapps/third_party_auth/management/commands/update_uid.py

```diff
@@ -0,0 +1,47 @@
+"""
+Base command to update the uid value on the in the UserSocialAuth table.
+"""
+from django.db import transaction
+from django.core.management.base import CommandError
+from social_django.models import UserSocialAuth
+
+from track.management.tracked_command import TrackedCommand
+
+
+class Command(TrackedCommand):
+    """
+    This command allows to update the uid using a value from the extra_data field
+    for a given provider.
+
+    If the given field doesn't exist, an exception will be raised and the process
+    won't finish.
+    """
+    help = """
+    This command updates all the uid fields in the UserSocialAuth table
+    taking a specific data from the extra_data field.
+
+    example:
+        manage.py ... update_uid google-oauth2 salesforceid
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('provider')
+        parser.add_argument('field')
+
+    @transaction.atomic
+    def handle(self, provider, field, *args, **kwargs):
```

Comment:
Unused argument args and kwargs. @andrey-canon

Reply:
the method  handle is called with more arguments, if I remove them, an exception is raised with the message `got an unexpected keyword argument ..........`

Reply:
Got it.